### PR TITLE
feat: add passive flag and alt to img

### PR DESCRIPTION
**Pull request by VitaliiDC8**

### CODE REVIEW

File: src/components/Image/Image.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const Image = (props: ImageProps) => {
     const projectSettings = useContext(ProjectSettingsContext);
     const {
         src,
-        alt,
+        alt = 'image alt',
```
Comment (yuberdysheva) [inline src/components/Image/Image.tsx:23]: Maybe we have to name alt consistent? I saw names started with Capital letters  and this one starts with lowercase letter

**Review by yuberdysheva — CHANGES_REQUESTED** [review]
I'm not sure about where we should storage names of frames and alts, maybe it has to be some i18n or variables? I want to check it with @gorgeousvlad

File: src/blocks/Slider/Slider.tsx [hunk 2]
```diff
@@ -156,7 +156,7 @@ export const SliderBlock = (props: WithChildren<SliderProps>) => {
     useEffect(() => {
         onResize();
 
-        window.addEventListener('resize', onResize);
+        window.addEventListener('resize', onResize, {passive: true});
```
[the remark this answers is not in the document]
Comment (VitaliiDC8) [reply src/blocks/Slider/Slider.tsx:159]: This requirements from Google Lighthouse 
https://developer.chrome.com/docs/lighthouse/best-practices/uses-passive-event-listeners/?utm_source=lighthouse&utm_medium=devtools

File: src/components/Button/Button.tsx [hunk 3]
```diff
@@ -69,7 +69,9 @@ const Button = (props: ButtonProps) => {
     }
 
     let icon;
-    let image = img && <img className={b('image')} src={buttonImg.url} alt={buttonImg.alt} />;
+    let image = img && (
+        <img className={b('image')} src={buttonImg.url} alt={buttonImg.alt || 'Button image'} />
```
[the remark this answers is not in the document]
Comment (VitaliiDC8) [reply src/components/Button/Button.tsx:73]: no it cannot be, buttonImg cannot be null only alt can be

File: src/components/Image/Image.tsx (hunk 1 above)
Comment (VitaliiDC8) [reply src/components/Image/Image.tsx]: done, Capital letter

**Review by yuberdysheva — APPROVED** [review]

**Pull request merged**